digital_twins/water_tank_FI_monitor/README.md

```diff
@@ -0,0 +1,52 @@
+The fault-injection plugin is an extension to the Maestro co-orchestration engine that enables injecting inputs and outputs of FMUs in an FMI-based co-simulation with tampered values.
```

Comment:
The updated README for **water-tank-FI** is at https://github.com/INTO-CPS-Association/DTaaS/tree/feature/distributed-demo/docs/user/examples/water_tank_FI. Please use this README and add NuRV-specific parts.